Assets/MixedRealityToolkit/Utilities/Editor/Setup/MixedRealityProjectConfigurator.cs

```diff
@@ -102,7 +102,9 @@ public bool IsActiveBuildTargetValid()
             { Configurations.VirtualRealitySupported, new ConfigGetter(() => { return XRSettingsUtilities.LegacyXREnabled; }) },
```

Comment:
Slightly unrelated, but I wonder if this line will cause issues with XR SDK enabled. I believe the legacy APIs always return `false` in that case.